plugins/hydra_ax_sweeper/example/conf/config.yaml

```diff
@@ -0,0 +1,33 @@
+defaults:
+  - hydra/sweeper: ax
+
+# Banana function
+# https://en.wikipedia.org/wiki/Rosenbrock_function
+banana:
+  # To be minimized
+  x: ???
+  y: ???
+  
+
+hydra:
+  ax:
+    experiment:
+      # Banana function should be minimized (Ax defaults to maximize)
+      minimize: true
+
+    early_stop:
+      # Number of epochs without a significant improvement from
+      # the currently known best parameters
+      # An Epoch is defined as a batch of trials executed in parallel
+      max_epochs_without_improvement: 10
+    
+    params:
+      banana.x:
+        type: range
+        bounds: [-5, 5]
+        log_scale: false
+      banana.y:
+        type: range
+        bounds: [-5, 10.1]
+        log_scale: false
```

Comment:
In this example, since we always want to use float for this function, we should probably use parameter_type to enforce it. (and tell the user what we are doing in a comment.

Reply:
`parameter_type` is not a required parameter (even as per Ax). Ax Plugin will promote types (wether launched via cmd or config) when it finds a mismatch (int mixed with float). If the user wants, they can specify `parameter_type`. I do not see the point of forcing this.

Reply:
Tacking in https://github.com/facebookresearch/hydra/issues/437